packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -947,6 +947,11 @@ Future<T?> showMenu<T>({
 /// Used by [PopupMenuButton.onSelected].
 typedef PopupMenuItemSelected<T> = void Function(T value);
 
+/// Signature for the callback invoked when a [PopupMenuButton] is opened.
+///
+/// Used by [PopupMenuButton.onOpened].
+typedef PopupMenuOpened = void Function();
```

Comment:
fly-by comment: Just use `VoidCallback` for this.

Reply:
The `onCanceled` callback is also a `VoidCallback` but it's using a similar definition:
```dart
typedef PopupMenuCanceled = void Function();
```
So I wanted to follow the same code style, should I use `VoidCallback` anyways?

Reply:
Yes, the `PopupMenu` should probably have used `VoidCallback` too.

Reply:
One note for posterity: we normally would prefer the present tense for function variables (and I just added that to the style guide, since it was missing), so `onOpen` instead of `onOpened`, but in this case it would be inconsistent with the rest of the API here, so I think you should keep it as `onOpened` instead of `onOpen`. It would be too much churn for too little improvement to deprecate the old names and create new ones just for the sake of consistency.

Reply:
Done! Now the `onOpened` callback is using a `VoidCallback`